api/openapi/main.hub.yml

```diff
@@ -62,4 +62,4 @@ pages:
           route:
             path: /api-reference
           data:
-            $ref: ./api.oas2.yml
+            $ref: ./solidus.yml
```

Comment:
I'd split this commit into a preparative one that move the yaml to the new format (maybe also a separate PR) and the actual change so that it is well documented.

Reply:
I slipped it in here to make the deprecation self consistent but it is an important change that needs its own PR 👍